antenna-documentation/scripts/documentation_release.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+
+set -e
+
+BASEDIR=$(dirname "$(readlink -f "$0")")
+
+CURRENT_TAG=${git describe --tag}
+
+createReleaseDocumentation() {
+  "${BASEDIR}"/execute_github_site_maven_plugin.sh
+}
+
+deleteOldSnapshotFolderInGhPages() {
+  fetchGhPages
+  git checkout gh-pages
+  OLD_SNAPSHOT_DIR=$(ls -d *SNAPSHOT*/)
+  rm -rf "*SNAPSHOT*"
+  git commit -am "Remove ${OLD_SNAPSHOT_DIR} documentation folder"
```

Comment:
This always generates a commit for the removal, even if nothing or not much was changed. This results in a long and possibly big git log / repo. Does this compromise the size or performance of our git repo?